GoogleDataTransport/GDTCORLibrary/Public/GoogleDataTransport/GDTCOREvent.h

```diff
@@ -74,6 +75,9 @@ typedef NS_ENUM(NSInteger, GDTCOREventQoS) {
 /** Bytes that can be used by an uploader later on. */
 @property(nullable, nonatomic) NSData *customBytes;
 
+/** The product data that the event is associated with, if any.  */
+@property(nullable, readonly, nonatomic) GDTCORProductData *productData;
```

Comment:
What's the reason for the productData class versus adding a productID property to GDTCorEvent?

Reply:
Discussed offline.